README.md

```diff
@@ -488,6 +488,28 @@ and the `GitHub` theme when in the _light mode_.
 alias cat="bat --theme=\$(defaults read -globalDomain AppleInterfaceStyle &> /dev/null && echo default || echo GitHub)"
 ```
 
+### Syntax of a file
+
+You can change the default syntax highlighting of the file using the `--map-syntax` option.
+It uses glob pattern matching and is matched on full path and filename.
+
+Use it like this:
```

Comment:
"You can use it like this:"

Small change, but adding "you can" changes the tone to be more welcoming and less instructional. I can't seem to explain why, though 😅

Reply:
Agreed. Done all the changes in the latest commit.